rules_building_block/defense_evasion_indirect_command_exec_pcalua_forfiles.toml

```diff
@@ -13,7 +13,7 @@ Identifies indirect command execution via Program Compatibility Assistant (pcalu
 """
 from = "now-119m"
 interval = "60m"
-index = ["logs-endpoint.events.*"]
+index = ["logs-endpoint.events.process-*"]
```

Comment:
this rule is compatilbe with sysmon and 4688, maybe not purpose of this PR but could be another one.

Reply:
++++